styx-common/src/test/java/com/spotify/styx/storage/AggregateStorageTest.java

```diff
@@ -53,6 +55,17 @@ public void setUp() throws Exception {
     sut = new AggregateStorage(bigtable, datastore);
   }
 
+  @After
+  public void tearDown() throws Exception {
+    sut.close();
```

Comment:
Does `sut` stand for `Storage Under Test`? :D

Reply:
`System Under Test`

http://xunitpatterns.com/SUT.html
https://stackoverflow.com/questions/7321407/what-is-sut-and-where-did-it-come-from
https://en.wikipedia.org/wiki/System_under_test